tools/kprint/main.cpp

```diff
@@ -16,7 +16,7 @@ cl::opt<std::string> DefinitionFilename(
 cl::opt<std::string>
     PatternFilename(cl::Positional, cl::desc("<pattern.kore>"), cl::Required);
 
-cl::opt<bool> FilterSubst(cl::Positional, cl::desc("[true|false]"));
+cl::opt<std::string> FilterSubst(cl::Positional, cl::desc("[true|false]"));
```

Comment:
This should be a `cl::opt<bool>`; doing so will save you from checking the string against the literal `"true"` later on.

Reply:
Fair enough, I'll change it back!